CollAction/Frontend/app/global/MapMarkerItem.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import renderComponentIf from "./renderComponentIf";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faMapMarkerAlt } from "@fortawesome/free-solid-svg-icons";
+
+library.add(faMapMarkerAlt);
+
+class MapMarkerItem extends React.Component {
```

Comment:
I'll leave it to @ljwboschker to the question whether this is necessary. Not sure myself.